netbox/ipam/api/views.py

```diff
@@ -79,7 +81,9 @@ class RoleViewSet(NetBoxModelViewSet):
 
 
 class PrefixViewSet(NetBoxModelViewSet):
-    queryset = Prefix.objects.all()
+    queryset = Prefix.objects.prefetch_related(
+        "scope",
+    ).all()
```

Comment:
The trailing `.all()` is unnecessary here (and below on `IPAddressViewSet.queryset`).

Reply:
removed

Reply:
Thanks, I'll get the review finished at some point today.